REQUIREMENTS.txt

```diff
@@ -4,7 +4,7 @@
 # installed, and bombs after the fact.
 python-dateutil < 2.8.1
 click
-dataclasses;python_version<"3.7"
+dataclasses;python_version==.6
```

Comment:
shouldn't this just be `dataclasses == 0.6`?

Reply:
yuuuup